Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -3337,7 +3762,7 @@ final class BuildPlanTests: XCTestCase {
                         .localSourceControl(path: .init("/drawPkg"), requirement: .upToNextMajor(from: "1.0.0")),
                     ],
                     products: [
-                        ProductDescription(name: "Game", type: .library(.automatic), targets: ["Utils"]),
+                        ProductDescription(name: "Game", type: .library(.automatic), targets: ["Game"]),
```

Comment:
this change is because the behavior is now more correct?

Reply:
This test is modified to cover more paths of alias merging